beacon-chain/db/kv/regen_historical_states.go

```diff
@@ -56,7 +57,10 @@ func (kv *Store) regenHistoricalStates(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	cacheState, err := lru.New(int(params.BeaconConfig().SlotsPerEpoch) * 2)
+	// Using max possible size to avoid using DB to save and retrieve pre state (slow)
+	// The size is 80 because block at slot 43772 built on top of block at slot 43693.
+	// That is the worst case.
+	cacheState, err := lru.New(80)
```

Comment:
Should this be a global constant? Just so its not hardcoded into a parameter.

Reply:
Yep! Done